src/views/Give/ManageDonationModal.tsx

```diff
@@ -126,15 +126,27 @@ export function ManageDonationModal({
       : theme.palette.text.secondary;
 
   const _useSohmBalance =
-    useSohmBalance()[activeChain.id == NetworkId.MAINNET ? NetworkId.MAINNET : NetworkId.TESTNET_RINKEBY];
+    useSohmBalance()[
+      activeChain.id == NetworkId.MAINNET
```

Comment:
Gross. Why are the ternary operations required? Why can't `activeChain.id` be passed? If you're checking whether it's an Ethereum chain, you could use the function I suggested earlier and if true, pass through `activeChain.id`.

Reply:
useSohmBalance returns a Record<NetworkId.MAINNET | NetworkId.TESTNET_RINKEBY | NetworkId.TESTNET_GOERLI, UseQueryResult<DecimalBigNumber, Error>>